Pyrado/scripts/training/qq-su_npdr.py

```diff
@@ -99,25 +99,36 @@
         policy = DummyPolicy(env_sim.spec)  # replaced by recorded real actions
 
     # Define a mapping: index - domain parameter
-    # dp_mapping = {0: "V_thold_neg", 1: "V_thold_pos"}
-    # dp_mapping = {0: "V_thold_neg", 1: "V_thold_pos", 2: "act_delay"}
-    # dp_mapping = {0: "Dr", 1: "Dp"}
-    # dp_mapping = {0: "Dr", 1: "Dp", 2: "Rm", 3: "km"}
-    # dp_mapping = {0: "Rm", 1: "km", 2: "Mr", 3: "Mp"}
-    # dp_mapping = {0: "Dr", 1: "Dp", 2: "Rm", 3: "km", 4: "Mr", 5: "Mp", 6: "Lr", 7: "Lp", 8: "g"}
-    dp_mapping = {0: "Dr", 1: "Dp", 2: "Rm", 3: "km", 4: "Mr", 5: "Mp", 6: "Lr", 7: "Lp", 8: "g", 9: "act_delay"}
+    # dp_mapping = {0: "voltage_thold_neg", 1: "voltage_thold_pos"}
+    # dp_mapping = {0: "voltage_thold_neg", 1: "voltage_thold_pos", 2: "act_delay"}
+    # dp_mapping = {0: "damping_rot_pole", 1: "damping_pend_pole"}
+    # dp_mapping = {0: "damping_rot_pole", 1: "damping_pend_pole", 2: "motor_resistance", 3: "motor_back_emf"}
+    # dp_mapping = {0: "motor_resistance", 1: "motor_back_emf", 2: "mass_rot_pole", 3: "mass_pend_pole"}
+    # dp_mapping = {0: "damping_rot_pole", 1: "damping_pend_pole", 2: "motor_resistance", 3: "motor_back_emf", 4: "mass_rot_pole", 5: "mass_pend_pole", 6: "length_rot_pole", 7: "length_pend_pole", 8: "gravity_const"}
```

Comment:
nice that you also got the commented stuff

Reply:
:)